src/components/navigation/navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import { slide as Menu } from 'react-burger-menu';
+import React from 'react';
+import PropTypes from 'prop-types';
+import './navbar.css';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import {
+  userIcon, scheculeIcon, discussionIcon, clipboardIcon, contactsIcon, clockIcon, signOutIcon,
+} from './navbarIcons';
+
+/* Styled Components */
+const SpacedDiv = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 8px 0;
```

Comment:
Make the padding a little larger (can be a little difficult to hit the right link on mobile

Reply:
I doubled the padding so this should be good now